assets/diagrams/sequence/figure65.plantuml

```diff
@@ -317,24 +317,24 @@ else
       data
     end hnote
     autonumber resume
-    PayerFSP ->> Switch: **GET /transactions/**<i><TransactionID></i>
+    PayerFSP ->> Switch: **GET /transactions/**<i><ID></i>
```

Comment:
Please leave it as it is, this is by design

Reply:
Okay, got it, reverted all similar changes.

Besides, in my new commit, I changed `PUT /participants/<ID>` to `PUT /participants/<TransactionID> in order to be consistent. Do you think it is okay or perhaps should I revert that again?

Reply:
Thanks @stevenwjy. For /participants it will be <ID> , please revert if needed

Reply:
Sorry, I don't get what you mean, should I revert it back to `PUT /participants/<ID>`?

Reply:
Yes, please

Reply:
Oh sorry... just realised that what I mean was `PUT /transactions/...`

However, when I was about to revert it, I realised that it would contradict with your other comments to change `PUT /transactions/<ID>` to `PUT /transactions/<TransactionID>` in figure 63. 

What I meant before was that I changed `PUT /transactions/<ID>` to `PUT /transactions/<TransactionID>` in figure 56, 64, and 65. Are you sure that you want to differentiate between figure 63 and the others?

If it is intended then I will revert my changes in figure 56, 64, and 65.

Reply:
Ok, if you changed it to `PUT /transactions/<TransactionID>`, then we should be fine..